### DISCUSSION TITLE
Eagle import function

### DISCUSSION THREAD

**Discussion by KannaCrossing** [TagStudioDev/TagStudio#170] [Ideas]
Since Eagle is a similar tool that the developer found but does not fit the exact needs, which I am the same, it would be good to have importing the tags as a potential feature.

**Comment by tadpole256** [comment]
I agree, the ability to import tags (generally) is a good idea.

**Comment by vg16855** [comment]
The Eagle Developers are making a Plugin and API system so that could work wonders.
